src/wp-admin/includes/file.php

```diff
@@ -2557,3 +2626,58 @@ function wp_opcache_invalidate( $filepath, $force = false ) {
 
 	return false;
 }
+
+/**
+ * Invalidate OPcache of directory of files.
+ *
+ * @since 6.2.0
+ *
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
+ *
+ * @param string $dir The path to invalidate.
+ *
+ * @return void
+ */
```

Comment:
```suggestion
/**
 * Attempts to clear the opcode cache for a directory of files.
 *
 * @since 6.2.0
 *
 * @see wp_opcache_invalidate()
 * @link https://www.php.net/manual/en/function.opcache-invalidate.php
 * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
 *
 * @param string $dir The path to the directory for which the opcode cache is to be cleared.
 */
```
Minor edits for consistency with `wp_opcache_invalidate()` and per the documentation standards.

Reply:
Oops, I'll have to fix this on my end.